.github/ISSUE_TEMPLATE/crash_report.md

```diff
@@ -0,0 +1,5 @@
+---
+title: WARNING crash found
```

Comment:
We should tag it as `bug`. Also, maybe add some tag specially for this.

Reply:
Resolved: https://github.com/lambdaclass/cairo-vm/pull/1331/commits/0720d307c39cec2c8638fcaede84009d452100cf